app/application/adapter.js

```diff
@@ -7,5 +7,17 @@ export default DS.RESTAdapter.extend({
 	namespace: 'api/v1',
 	headers: {
 		'Authorization': 'Token token=' + ENV.datastoreAuthToken
-  }
-});
+	},
+	ajaxOptions: function(url, type, options) {
+		var hash = this._super(url, type, options);
+		if (type === 'GET') {
+			let data = {};
+			if (typeof(hash.data) === 'string') {
+				data = JSON.parse(hash.data);
+			}
+			data["per_page"] = 5;
```

Comment:
@drewda I wanted to keep this in here (on dev only) until the store goes over 50 operators.